pages/common/spatial.md

```diff
@@ -0,0 +1,27 @@
+# spatial
+
+> A set of commands for managing and developing SpatialOS projects.
+
+- Build workers for local deployment on Unity on Windows:
+
+`spatial worker build --target=development --target=Osx`
+
+- Build workers for local deployment on Unreal:
+
+`spatial worker build --target=local --target=Windows`
```

Comment:
Here your target is local, but in the previous example, it was development and still the description says "local deployment". Are the two not related ?

Reply:
Unreal and Unity use different targets

Reply:
My question was how does specifying the target change the deployment type. Here target=local, and you say "local deployment", which is fine. But above this, you have "target=development" but still it was "local deployment". 

So what does development and local targets mean when both of them are used for local deployments ?

Reply:
Hey! Both targets build for local development, but when building for Unreal you use `--target=local` and when building for Unity you use `--target=development`

I flagged this to the devs

Reply:
I see. That is very confusing indeed.